luaui/Widgets/gui_healthbars_gl4.lua

```diff
@@ -350,6 +350,13 @@ local minReloadTime = 4 -- weapons reloading slower than this willget bars
 local featureHealthVBO
 local featureResurrectVBO
 local featureReclaimVBO
+
+local minUiScale = 0.8    -- FIXME: Get these values from "Options" widget, respectively min and max value from 'uiscale'
+local maxUiScale = 1.15
+local nMinScale = 0.9     -- Transpose and scale linearly default scale for healthbars
```

Comment:
Adjusting these two values allow for optimization of the scaling for healthbars in relation to ui_scale